data/wanderers middle.txt

```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			choice
+				`	"How did your ship end up in such a state?"`
+				`	"Was this beauty the victim of a pirate attack?"`
+				
+			`	"I'm not quite sure," the captain replies. "We were shot by a fleet of hostile ships that our computer couldn't identify. They didn't look at all like a run-of-the-mill pirate ship, though, and they were kitted out with tech I've never seen before."`
+			choice
+				`	"How many ships were there? What did they look like?"`
+				`	"Where did this fight take place?"`
+					goto where
+				`	"You should be glad you're still in one piece."`
+					goto glad
+					
+			`	"Well," the pilot continues, "we had hired a beefy Leviathan escort to Farpoint in the Alnitak system. When we were going in for landing, two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies in this area. The ships weren't big, but they were tough——our four heavy laser turrets weren't enough to produce even a dent in their shields. They made short work of the Leviathan, and nearly disabled our ship, too. It was lucky I decided that jumping would be faster than landing. We warped out of the system while in atmosphere, saving the ship, but now I'm in debt to all my crew because I couldn't meet my rush delivery deadline."`
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm looking for."`
+					goto thanks
+					
+			label where
+			`	"Well," the pilot explains, "we were doing a large cargo run to Farpoint in the Alnitak system, when two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies in this area. I had to jump the ship in atmosphere to stop us from getting shredded by their strange melting weaponry. I immediately reported the attack to a desk clerk at the Navy base in the neighboring system Saiph, but their servers show no logs of any unidentified ships beyond Farpoint, so I guess they turned around after scouting out the Alnitak system."`
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm hunting for."`
+					goto thanks
+			
+			label thanks
+			`	"No problem, kiddo," the captain says, although he looks awed that you are even attempting to fight such powerful ships. "Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; those are some dangerous ships that jumped us." With that, he bids you farewell.`
+					decline
+					
+			label glad
+			`	"Yeah, those ships would have obliterated us if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those ships?"`
```

Comment:
Are you sure it is worth using "quizzically"? I'm pretty sure not many people, starting with me, know this word, and [Dictionary.com](http://www.dictionary.com/browse/quizzically?s=ts) agrees.